crates/wasi-crypto/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "wasmtime-wasi-crypto"
+version = "0.22.0"
+authors = ["The Wasmtime Project Developers"]
+description = "Wasmtime implementation of the wasi-crypto API"
+documentation = "https://docs.rs/wasmtime-wasi-crypto"
+license = "Apache-2.0 WITH LLVM-exception"
+categories = ["wasm", "cryptography"]
+keywords = ["webassembly", "wasm", "crypto"]
+repository = "https://github.com/bytecodealliance/wasmtime"
+readme = "README.md"
+edition = "2018"
+
+[dependencies]
+anyhow = "1.0"
+wasi-crypto = "0.1.3"
```

Comment:
Is this intended to be a `path` dependency into the submodule perhaps?

Reply:
`path` dependency added!

Reply:
That requires a change to the `publish` script. Done.